.nvmrc

```diff
@@ -0,0 +1 @@
+v18
```

Comment:
I do not use `nvm` and I am not smart enough to remember that when the lowest version of node required for this project changes in the future (when I change `engine.node` in package.json), I need to change `.nvmrc` at the same time.
Therefore, I do not want `.nvmrc`, `.node-version`, etc. to be included in the project.

Reply:
Ok, I'm removing it then :)